en/mkdocs.yml

```diff
@@ -2112,5 +2113,5 @@ extra:
         - type: linkedin
           link: https://www.linkedin.com/company/wso2
     site_version: 4.0.0
-    #base_path: http://localhost:8000
-    base_path: https://apim.docs.wso2.com/en/4.0.0
+    base_path: http://localhost:8000
+    #base_path: https://apim.docs.wso2.com/en/4.0.0
```

Comment:
Let's uncomment the base path and comment localhost.

Reply:
Fixed.